sonoff.py

```diff
@@ -35,14 +31,13 @@ def login(self):
             'password': self.password
         }
         hex_dig = hmac.new(self.appsecret.encode(),
-                                json.dumps(payload).encode(),
-                                digestmod=hashlib.sha256).digest()
+                           json.dumps(payload).encode(),
```

Comment:
in general I prefer to separate between code refactoring commits and new features commits as it make future debugging more difficult. In this case you did very small refactoring changes, so it is ok. thanks.